### PR TITLE
Added position control (ucrSave) to the calculator

### PR DESCRIPTION
**Pull request by Wycklife**
This pull request is not yet ready for review. There are some on going discussions between @Wycklife  and @lloyddewit .

@lloyddewit You can now get these changes.

### CODE REVIEW

File: instat/dlgCalculator.vb [hunk 1]
```diff
@@ -80,30 +50,42 @@ Public Class dlgCalculator
         ucrBase.clsRsyntax.AddToBeforeCodes(clsAttach)
         ucrBase.clsRsyntax.AddToAfterCodes(clsDetach)
         ucrBase.clsRsyntax.SetCommandString("")
-        ucrCalc.ucrSaveResultInto.SetItemsTypeAsColumns()
-        ucrCalc.ucrSaveResultInto.SetDefaultTypeAsColumn()
+        ucrCalc.ucrSaveResultInto.SetSaveTypeAsColumn()
+        ucrCalc.ucrSaveResultInto.SetCheckBoxText("Save Result Into")
+        ucrCalc.ucrSaveResultInto.SetDataFrameSelector(ucrCalc.ucrSelectorForCalculations.ucrAvailableDataFrames)
```
Comment (lloyddewit) [inline instat/dlgCalculator.vb:55]: This line duplicates the line below

**Comment by Patowhiz** [comment]
@Wycklife once this is ready for review please let me know. I may suggest more enhancements to to the `ucrSave `control in addition to what I did to it previously, depending on how it will be used in this dialog.

**Comment by Wycklife** [comment]
@Patowhiz Why don't you take this up if at you're working on ucrSave control? There are some dialogs  with this control but it doesn't work. So you can implement this then once you're done, I can work on other dialogs.

**Comment by lloyddewit** [comment]
@Patowhiz I'm currently commenting ucrSave (see PR #5794). I hope to be finsihed this week. If you plan to change the ucrSave functionality then please let me know so that we can avoid merge conflicts. Thanks

**Comment by Patowhiz** [comment]
@Wycklife @lloyddewit  I'm not working on the ucrSave control, I mean't I wanted to see how you will use it in the dialog (It's one of the special dialogs), so that I can suggest any enhancements that could remove any duplications when the control is added in other special dialogs.

@Wycklife I'm happy to help with any implementation of this dialog to take advantage of the control I can see from the comment above that there are discussions between you and @lloyddewit, I'm happy to jump in and help if necessary.

If there are other dialogs that have the control and are not working, please outline them (probably in another issue) so that I can quickly check and see what could be the problem.

**Comment by lloyddewit** [comment]
@Wycklife As far as I know, this enhancement is still required. Please could you clarify the reason for closing this PR? Will you replace this PR with another? thanks

**Comment by Wycklife** [comment]
@lloyddewit This was closed because @Patowhiz took it up and he is working on it.

**Comment by lloyddewit** [comment]
@Wycklife, @Patowhiz I discussed with @rdstern. I understand that @volloholic prefers @Patowhiz to work on some other tasks. 

@Wycklife Please could you take over this PR again? thanks

**Comment by Wycklife** [comment]
@lloyddewit Okay, I have taken it up.

**Pull request closed without merging**